pbrd/pbr_map.c

```diff
@@ -268,7 +268,7 @@ struct pbr_map_sequence *pbrms_get(const char *name, uint32_t seqno)
 	pbrm = pbrm_find(name);
 	if (!pbrm) {
 		pbrm = XCALLOC(MTYPE_PBR_MAP, sizeof(*pbrm));
-		strcpy(pbrm->name, name);
+		snprintf(pbrm->name, sizeof(pbrm->name), "%s", name);
```

Comment:
Even though we don't do return checks, I think `strlcpy` fits better (and the rest of the occurrences in this file).

Reply:
functionally equivalent